### PR TITLE
Don't start backfiller until after migrations

### PR DESCRIPTION
**Pull request by phildarnowsky**

### CODE REVIEW

**Comment by phildarnowsky** [comment]
@lemald note that I'm planning to override codecov's complaints here, you'll see why if you look at the report, and this code is temporary anyway.

File: lib/skate/application.ex [hunk 1]
```diff
@@ -34,6 +34,10 @@ defmodule Skate.Application do
 
     Migrate.up()
 
+    if Application.get_env(:skate, :start_data_processes) do
+      send(Notifications.BlockWaiverBackfiller.default_name(), :backfill_batch)
+    end
```
Comment (lemald) [inline lib/skate/application.ex:39]: So, one alternative that comes to mind, which might be simpler and not require any overriding of codecov, would be to simply remove the code that starts the `Notifications.BlockWaiverBackfiller` process, deploy that, and then re-add it and deploy it once the previous version has gone out and the migration has passed. Do you have any thoughts on the relative merits of that?

Comment (phildarnowsky) [reply lib/skate/application.ex:39]: @lemald that would also work, and we were indeed talking about the merits of deploying migrations on their own, but in this case I'd rather save the extra deploy, especially as this code is coming back out again very soon anyway, so I'm not as concerned about a little hackishness.

Comment (phildarnowsky) [reply lib/skate/application.ex:39]: @lemald I definitely think that, going forward though, it would be a good practice to migrate separately.

**Review by lemald — APPROVED** [review]
👍

File: lib/skate/application.ex (hunk 1 above)
Comment (phildarnowsky) [reply lib/skate/application.ex:39]: @lemald scratch that--you can't override the codecov patch check after all. I'll go ahead and do a new PR with just the migrations.

**Pull request closed without merging**